dist/style.css

```diff
@@ -59,3 +55,288 @@ body {
+  margin-top: 5%;
+  width: 100%;
+  justify-content: space-between;
+}
+
+.container-moto {
+  font-family: 'AlexandriaFLF';
+  display: flex;
+  flex-direction: row;
+  flex-wrap: nowrap;
+  width: 100%;
+  justify-content: left;
+}
+
+/* effects for hover on buttons */
+.menu {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: nowrap;
+  justify-content: space-between;
+  background-color: white;
+  text-align: center;
+  padding: 2%;
+  border-radius: 25px;
+  border: 2px solid #A9A9A9;
+  width: 25%;
+}
+
+.menu:hover {
+  background-color: #662A61;
+  color: white;
+}
+
+a {
+  text-decoration: none;
+}
+
+.fas:hover {
+  background-color: #562554;
+  color: white;
+}
+
+p {
+  margin: 0px;
+  margin-left: 7.5%;
+  margin-right: 10.5%;
+}
+
+fas {
+  height: 0%;
+}
+
+/* logo resizing  */
+.container-logo {
+  max-width: 65%;
+  height: auto;
+}
+
+#mainlogo {
+  max-width: 65%;
+  height: auto;
+  padding-right: -25%;
+}
+
+
+/* main content in split boxes  */
+.container-body {
+  margin-bottom: 20px;
+}
+
+.content {
+  font-family: 'AlexandriaFLF';
+  display: flex;
+  flex-direction: row;
+  border: 5px solid #562554;
+  border-radius: 25px;
+  background-color: white;
+}
+
+.content-left {
+  display: flex;
+  flex-direction: column;
+  margin: 2%;
+  width: 50%;
+  align-items: center;
+}
+
+.content-right {
+  display: flex;
+  flex-direction: column;
+  margin: 2%;
+  width: 50%;
+  align-items: center;
+  text-align: center;
+}
+
+/* form styling  */
+input {
+  border-radius: 25px;
+  width: 100%;
+  background-color: #336D61;
+  color: white;
+  padding: 6.5px;
+  margin-top: 4px;
+  margin-bottom: 1.5px;
+}
+
+::placeholder {
+  color: white;
+  opacity: 1;
+}
+
+.disclaimer {
+  font-family: 'Courier New', Courier, monospace;
+  font-size: 75%;
+  color: rgb(88, 87, 87);
+  text-align: left;
+  padding-right: 5px;
+}
+
+#msg {
+  height: 200px;
+  border-radius: 25px;
+  width: 100%;
+  background-color: #336D61;
+  color: white;
+  margin-bottom: 1.5px;
+  margin-top: 6px;
+  padding-left: 6.5px;
+  padding-top: 16.5px;
+}
+
+/* disclaimer and send button  */
+.container-send {
+  display: flex;
+  flex-direction: row;
+}
+
+.send {
+  background-color: #336D61;
+  color: white;
+  text-align: center;
+  padding: 5px 8px;
+  border-radius: 25px;
+  border: 2px solid #A9A9A9;
+  padding: 5px;
+  height: 40px;
+  width: 320px;
+  margin-top: 2%;
+}
+
+.send:hover {
+  background-color: #562554;
+  color: white;
+}
+
+/* left side content styling */
+.social {
+  display: flex;
+  flex-direction: row;
+  font-size: 3em;
+  justify-content: space-between;
+  margin-top: 7%;
+  margin-bottom: 6%;
+  width: 50%;
+}
+
+.share {
+  width: 50%;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+hr {
+  width: 60%;
+  margin-left: auto;
+  margin-right: auto;
+}
+
+.ownerTitle{
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  font-family: AlexandriaFLF-Bold;
+  font-size: 1.4em;
```

Comment:
- Not sure if we should vary the `font-size` here...I think it would be better to only change the font-size when we hit a breakpoint. Let's handle responsiveness with this text once we start considering tablet and mobile views, and make the font a static size for now.

- Can we make it bigger, to match the mockup?

Reply:
Same applies to all other text in this area of the page. (Regarding responsive `font-size`)